### ISSUE TITLE
Everything worked fine two days back but now I am getting an error Uncaught type error

### ISSUE THREAD

**Issue by rahulss14** [aws-samples/amazon-sumerian-hosts#43]
<img width="513" alt="Screenshot 2021-04-28 at 12 58 35 AM" src="https://user-images.githubusercontent.com/45919225/116301052-3aace900-a7bd-11eb-9e4f-ff50669d745b.png">

**Comment by ivanfarkas2** [comment]
Yes. I am getting the same thing. It never worked for me since I've just started exploring.
It fails in [amazon-sumerian-hosts/src/core/Messenger.js](https://github.com/aws-samples/amazon-sumerian-hosts/blob/9984e3625965c52eadd887c494963382dd29a044/src/core/Messenger.js#L99)

```JavaScript
this._dispatcher.addEventListener(
  this._createLocalMessage(message),
  listener
);
```

Get exception:
```
Uncaught (in promise) TypeError: this._dispatcher.addEventListener is not a function
    at HostObject._addListener (333:formatted:1841)
    at HostObject.listenTo (333:formatted:1880)
    at new FeatureDependentMixin (333:formatted:3532)
    at LipsyncFeature._createSuperInternal (333:formatted:4802)
    at new TextToSpeechFeatureDependentMixin (333:formatted:4989)
    at LipsyncFeature._createSuperInternal (333:formatted:3374)
    at new FeatureDependentMixin (333:formatted:3521)
    at LipsyncFeature._createSuperInternal (333:formatted:3757)
    at new AnimationFeatureDependentMixin (333:formatted:3959)
    at LipsyncFeature._createSuperInternal (333:formatted:4241)
```
![Error](https://user-images.githubusercontent.com/53225260/116820453-3f69f800-ab43-11eb-8ea2-a8e37a6a73b7.png)

**Comment by IvanFarkas** [comment]
Yes. I am getting the same thing. It never worked for me since I've just started exploring.
It fails in [amazon-sumerian-hosts/src/core/Messenger.js](https://github.com/aws-samples/amazon-sumerian-hosts/blob/9984e3625965c52eadd887c494963382dd29a044/src/core/Messenger.js#L99)

```JavaScript
this._dispatcher.addEventListener(
  this._createLocalMessage(message),
  listener
);
```
message='onAddFeature'

Get exception:
```
Uncaught (in promise) TypeError: this._dispatcher.addEventListener is not a function
    at HostObject._addListener (333:formatted:1841)
    at HostObject.listenTo (333:formatted:1880)
    at new FeatureDependentMixin (333:formatted:3532)
    at LipsyncFeature._createSuperInternal (333:formatted:4802)
    at new TextToSpeechFeatureDependentMixin (333:formatted:4989)
    at LipsyncFeature._createSuperInternal (333:formatted:3374)
    at new FeatureDependentMixin (333:formatted:3521)
    at LipsyncFeature._createSuperInternal (333:formatted:3757)
    at new AnimationFeatureDependentMixin (333:formatted:3959)
    at LipsyncFeature._createSuperInternal (333:formatted:4241)
```
![Error](https://user-images.githubusercontent.com/53225260/116820453-3f69f800-ab43-11eb-8ea2-a8e37a6a73b7.png)

![call_stack](https://user-images.githubusercontent.com/53225260/116820614-fcf4eb00-ab43-11eb-84b8-49090599c705.png)

**Comment by rahulss14** [comment]
Thank you so much @IvanFarkas for the fix . Were you able to find a fix for three.js r128 ?

**Comment by IvanFarkas** [comment]
Your welcome @rahulss14.
I do not have a fix for three.js r128 and ideally this should come from the Amazon or the repo maintainers.
Let's see how quick they do.
If not, I put my team to work in a month.

**Comment by jkerste** [comment]
Thanks for reporting this issue @rahulss14, and thanks for providing a workaround @IvanFarkas. This issue should be fixed in v1.3.5, please re-open if you continue to have issues after getting the latest release.
